### PR TITLE
refactor: update text input class component to functional

### PR DESCRIPTION
**Pull request by unikvozm**
### Summary

TextInput class components are rewritten as functional ones.

### CODE REVIEW

File: src/components/TextInput/Adornment/Affix.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { AdornmentSide } from './enums';
 
 const AFFIX_OFFSET = 12;
 
-type Props = {
+export type Props = {
```
Comment (matkoson) [inline src/components/TextInput/Adornment/Affix.tsx:17]: Those exports are related to some other PR? It seems to not be used in this one.

Comment (unikvozm) [reply src/components/TextInput/Adornment/Affix.tsx:17]: They're used by `CompoundedComponent` inside TextInput.tsx

**Review by matkoson — APPROVED** [review]
lgtm

**Review by jaworek — APPROVED** [review]

File: src/components/TextInput/TextInputFlat.tsx [hunk 2]
```diff
@@ -47,353 +47,331 @@ const MIN_HEIGHT = 64;
...
-              adornmentStyleAdjustmentForNativeInput,
-            ],
-          })}
-        </View>
-        <TextInputAdornment {...adornmentProps} />
+            ]}
+          />
+        )}
+        <InputLabel parentState={parentState} labelProps={labelProps} />
+        {render?.({
```
Comment (jaworek) [inline src/components/TextInput/TextInputFlat.tsx:334]: It would be good to replace rendering of custom input component with JSX instead of function call.

Comment (lukewalczak) [reply src/components/TextInput/TextInputFlat.tsx:334]: That makes sense, let's cover your idea in the next iteration, what will be definitely easier to review.

**Pull request merged**